---
abstract: CRISPR technology, combined with single-cell RNA-Seq, has opened the way
  to large scale pooled perturbation screens, allowing more systematic interrogations
  of gene functions in cells at scale. However, such Perturb-seq data poses many analysis
  challenges, due to its high-dimensionality, high level of technical noise, and variable
  Cas9 efficiency. The single-cell nature of the data also poses its own challenges,
  as we observe the heterogeneity of phenotypes in the unperturbed cells, along with
  the effect of the perturbations. All in all, these characteristics make it difficult
  to discern subtler effects. Existing tools, like mixscape and ContrastiveVI, provide
  partial solutions, but may oversimplify biological dynamics, or have low power to
  characterize perturbations with a smaller effect size. Here, we address these limitations
  by introducing the Supervised Contrastive Variational Autoencoder (SC-VAE). SC-VAE
  integrates guide RNA identity with gene expression data, ensuring a more discriminative
  analysis, and adopts the Hilbert-Schmidt Independence Criterion as a way to achieve
  disentangled representations, separating the heterogeneity in the control population
  from the effect of the perturbations. Evaluation on large-scale data sets highlights
  SC-VAE’s superior sensitivity in identifying perturbation effects compared to ContrastiveVI,
  scVI and PCA. The perturbation embeddings better reflect known protein complexes
  (evaluated on CORUM), while its classifier offers promise in identifying assignment
  errors and cells escaping the perturbation phenotype. SC-VAE is readily applicable
  across diverse perturbation data sets.
title: A Supervised Contrastive Framework for Learning Disentangled Representations
  of Cellular Perturbation Data
layout: inproceedings
series: Proceedings of Machine Learning Research
publisher: PMLR
issn: 2640-3498
id: tu24a
month: 0
tex_title: A Supervised Contrastive Framework for Learning Disentangled Representations
  of Cellular Perturbation Data
firstpage: 90
lastpage: 100
page: 90-100
order: 90
cycles: false
bibtex_author: Tu, Xinming and H\"utter, Jan-Christian and Wang, Zitong Jerry and
  Kudo, Takamasa and Regev, Aviv and Lopez, Romain
author:
- given: Xinming
  family: Tu
- given: Jan-Christian
  family: Hütter
- given: Zitong Jerry
  family: Wang
- given: Takamasa
  family: Kudo
- given: Aviv
  family: Regev
- given: Romain
  family: Lopez
date: 2024-03-15
address:
container-title: Proceedings of the 18th Machine Learning in Computational Biology
  meeting
volume: '240'
genre: inproceedings
issued:
  date-parts:
  - 2024
  - 3
  - 15
pdf: https://proceedings.mlr.press/v240/tu24a/tu24a.pdf
extras: []
# Format based on Martin Fenner's citeproc: https://blog.front-matter.io/posts/citeproc-yaml-for-bibliographies/
---
